packages/modal/README.md

```diff
@@ -188,3 +188,78 @@ function AlertDialogExample() {
   )
 }
 ```
+
+# Drawer
+
+The Drawer component is a panel that slides out from the edge of the screen. 
+It can be useful when you need users to complete a task or view some details 
+without leaving the current page.
+
+## Installation
+
+```sh
+yarn add @chakra-ui/drawer
+
+# or
+
+npm i @chakra-ui/drawer
+```
+
```

Comment:
Since the drawer component is included in the package `@chakra-ui/modal` it is no longer published as standalone package. 

Kindly replace `@chakra-ui/drawer` with `@chakra-ui/modal`.